Makefile

```diff
@@ -34,7 +34,7 @@ requirements-dev: virtualenv test-requirements requirements-dev.txt
 .PHONY: freeze-requirements
 freeze-requirements:
 	rm -rf venv-freeze
-	virtualenv -p python3 venv-freeze
+	python3 -m venv venv-freeze
 	$$(pwd)/venv-freeze/bin/pip install -r requirements-app.txt
```

Comment:
hmm for me, `venv` doesn't automatically install `pip`, leading this line to then fail.

Reply:
This might be related to [this Travis error](https://travis-ci.org/alphagov/digitalmarketplace-frontend-toolkit/builds/422511041?utm_source=github_status&utm_medium=notification); apparently Debian systems have `python-venv` as a separate package.